content/docs/instrumenting/clientlibs.md

```diff
@@ -21,6 +21,7 @@ HTTP endpoint on your application’s instance:
 Unofficial third-party client libraries:
 
 * [Bash](https://github.com/aecolley/client_bash)
+* [Clojure](https://github.com/soundcloud/prometheus-clj) (wraps the Java client library)
```

Comment:
The "directly instrumented software" section is perhaps a bit misnamed, as gokit and go-metrics don't really fit. So perhaps we should move them too to the new section, which we could name in a very broad way to not get into discussions what's a wrapper and what's not.